### PR TITLE
BC-2529 - global css

### PR DESCRIPTION
**Pull request by christian-darsow**
# Short Description

Reduce build css size by using global css

## Links to Ticket and related Pull-Requests
https://ticketsystem.dbildungscloud.de/browse/BC-2529

## Checklist before merging

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] PO: Any deviation from requirements was agreed with Product-Owner / ticket author / support-team

> Notice: Please keep this Pull-Request as a Draft (or add WIP label), until it is ready to be reviewed

### CODE REVIEW

File: src/pages/RoomDetails.page.vue [hunk 1]
```diff
@@ -397,7 +397,6 @@ export default defineComponent({
 </script>
 <style lang="scss" scoped>
 @import "~vuetify/src/styles/styles.sass";
```
Comment (OliverHappe) [inline src/pages/RoomDetails.page.vue:399]: i think we should not do this. 

in this component we need the breakpoint mixin from vuetify.

i suggest to use the breakpoint display classes in the template in favor of importing all vuetify styles just for a mixin.

alternatively: is it possible to just import the mixin?

Comment (christian-darsow) [reply src/pages/RoomDetails.page.vue:399]: this vuetify import could be removed from some components complety as none of the imported functionality was used. for the others I would say we can stay with this import, as it only imports mixins / vars and nothing that gets compiled into the css. I would say that this is also with the docs https://vuetifyjs.com/en/features/sass-variables/#usage-in-templates

File: src/pages/RoomDetails.page.vue [hunk 2]
```diff
@@ -397,7 +397,6 @@ export default defineComponent({
...
-@import "@variables";
```
[the remark this answers is not in the document]
Comment (christian-darsow) [reply src/pages/RoomDetails.page.vue:400]: goid point. unneeded imports are removed from several components now

File: src/themes/base/styles/index.scss [hunk 3]
```diff
@@ -1,2 +1,13 @@
-@import "./utility.scss";
```
[the remark this answers is not in the document]
Comment (christian-darsow) [reply src/themes/base/styles/index.scss]: the index.scss from the themes now import it as mixins from the base

File: src/pages/RoomDetails.page.vue (hunk 1 above)
Comment (OliverHappe) [reply src/pages/RoomDetails.page.vue:399]: thanks for the clarification!

**Review by odalys-dataport — APPROVED** [review]

**Pull request merged**